catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/transformer/TransformerDescriptorApplication.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.transformer;
+
+import static org.codice.ddf.catalog.ui.transformer.TransformerDescriptorUtilities.getTransformerDescriptor;
+import static org.codice.ddf.catalog.ui.transformer.TransformerDescriptorUtilities.getTransformerDescriptors;
+import static spark.Spark.get;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.osgi.framework.ServiceReference;
+import spark.servlet.SparkApplication;
+
+public class TransformerDescriptorApplication implements SparkApplication {
+
+  private static final Gson GSON =
+      new GsonBuilder().disableHtmlEscaping().serializeNulls().create();
+
+  private List<Map<String, String>> metacardTransformers = Collections.emptyList();
+
+  private List<Map<String, String>> queryResponseTransformers = Collections.emptyList();
+
+  private static final Map<String, String> NOT_FOUND_RESPONSE =
+      new ImmutableMap.Builder<String, String>().put("message", "Transformer not found").build();
+
+  @Override
+  public void init() {
+    get("/transformers/metacard", (req, res) -> GSON.toJson(metacardTransformers));
+
+    get(
```

Comment:
✏️ This action and `/transformers/query/:id` are identical save for the transformers they pass to the `getTransformerDescriptor`. Should probably encapsulate this functionality into a method that takes transformers as a parameter